src/svgp.jl

```diff
@@ -1,10 +1,13 @@
-struct SVGP end
+struct SVGP{Tfz<:FiniteGP,Tq<:AbstractMvNormal}
+    fz::Tfz
```

Comment:
General question (also @willtebbutt): how would this work with inter-domain approximations ? Would it simply be a different `FiniteGP` instance ?

Reply:
My inclination would just be to make `z` the inputs associated with the interdomain features. i.e. if you're placing pseudo-points in a convolved version of your original process, write that down in Stheno.jl and put `z` in the convolved process.

Reply:
and e.g. if you're doing Fourier features or spherical harmonics (where "z" lives in a completely different space, e.g. some kind of discrete set indexing the frequencies)?